apps/evans/components/core/AudioPlayer.tsx

```diff
@@ -46,10 +67,17 @@ const AudioPlayer: React.FC<Props> = ({ tracks }) => {
   useEffect(() => {
     if (audio) {
       audio.addEventListener(`timeupdate`, () => {
+        localStorage.setItem(
```

Comment:
let's also extract a function to write this data, that takes the path, and the two numbers, to clean up the call site here a hair, and also abstract away the details of creating the key for both read and write.